ouroboros-network/src/Ouroboros/Network/Diffusion/Configuration.hs

```diff
@@ -0,0 +1,109 @@
+{-# LANGUAGE NamedFieldPuns #-}
+
+module Ouroboros.Network.Diffusion.Configuration
+  ( module Ouroboros.Network.Diffusion.Configuration
+    -- re-exports
+  , AcceptedConnectionsLimit (..)
+  , BlockFetchConfiguration (..)
+  , ChainSyncTimeout (..)
+  , MiniProtocolParameters (..)
+  , P2P (..)
+  , PeerSelectionTargets (..)
+  , PeerSharing (..)
+  , defaultMiniProtocolParameters
+  , deactivateTimeout
+  , closeConnectionTimeout
+  , peerMetricsConfiguration
+  , defaultTimeWaitTimeout
+  , defaultProtocolIdleTimeout
+  , defaultResetTimeout
+  , handshake_QUERY_SHUTDOWN_DELAY
+  , ps_POLICY_PEER_SHARE_STICKY_TIME
+  , ps_POLICY_PEER_SHARE_MAX_PEERS
```

Comment:
@bolt12 are these values used in `cardano-node` or are these only internals.

Reply:
They are used in `ouroboros-consensus`